copy/manifest.go

```diff
@@ -127,14 +127,14 @@ func isMultiImage(ctx context.Context, img types.UnparsedImage) (bool, error) {
 // forced value, and returns the MIME type to which we should convert the list
 // of manifests, whether we are converting to it or using it unmodified.
 func (c *copier) determineListConversion(currentListMIMEType string, destSupportedMIMETypes []string, forcedListMIMEType string) (string, error) {
-	// If we're forcing it, we prefer the forced value over everything else.
-	if forcedListMIMEType != "" {
-		return forcedListMIMEType, nil
```

Comment:
(Aside from adding debug output, this has the side effect of rejecting unrecognized `forcedListMIMEType` values. That’s not bad, just surprising given the commit description.)

Reply:
Hmm, yeah, I'll take another look at this bit.

Reply:
I’m sorry, to be clear, I’m fine with the change itself — it would fail anyway, and this way it fails before copying all of the images. It’s just the mismatch with the description that surprised me.

Reply:
Oh, okay.  I'd switched that up a bit, but I think I can change it back and fix the comment and commit log.